charts/sn-platform/templates/bookkeeper/_autorecovery.tpl

```diff
@@ -63,6 +64,52 @@ ad.datadoghq.com/{{ template "pulsar.autorecovery.podName" . }}.instances: |
     }
   ]
 {{- end }}
+{{- if eq .Values.datadog.adVersion "v2" }}
+ad.datadoghq.com/{{ template "pulsar.autorecovery.podName" . }}.checks: |
+  {
+    "openmetrics": {
+      "init_config": [{}],
+      "instances": [
+    {
+      "prometheus_url": "http://%%host%%:{{ .Values.autorecovery.ports.http }}/metrics",
+      {{ if .Values.datadog.namespace -}}
+      "namespace": "{{ .Values.datadog.namespace }}",
+      {{ else -}}
+      "namespace": "{{ template "pulsar.namespace" . }}",
+      {{ end -}}
+      "metrics": {{ .Values.datadog.components.autorecovery.metrics }},
+      "health_service_check": true,
+      "prometheus_timeout": 1000,
+      "max_returned_metrics": 1000000,
+      "type_overrides": {
+        "jvm_memory_bytes_used": "gauge",
+        "jvm_memory_bytes_committed": "gauge",
+        "jvm_memory_bytes_max": "gauge",
+        "jvm_memory_bytes_init": "gauge",
+        "jvm_memory_pool_bytes_used": "gauge",
+        "jvm_memory_pool_bytes_committed": "gauge",
+        "jvm_memory_pool_bytes_max": "gauge",
+        "jvm_memory_pool_bytes_init": "gauge",
+        "jvm_memory_direct_bytes_used": "gauge",
+        "jvm_threads_current": "gauge",
+        "jvm_threads_daemon": "gauge",
+        "jvm_threads_peak": "gauge",
+        "jvm_threads_started_total": "gauge",
```

Comment:
The metrics with the `_total` subfix are counter-type. https://github.com/DataDog/integrations-core/blob/master/openmetrics/datadog_checks/openmetrics/data/conf.yaml.example#L97-L101
> Note: To collect counter metrics with names ending in `_total`, specify the metric name without the `_total`
> suffix. For example, to collect the counter metric `promhttp_metric_handler_requests_total`, specify
> promhttp_metric_handler_requests`. This submits to Datadog the metric name appended with `.count`.

Reply:
fixed in https://github.com/streamnative/charts/pull/1199/commits/76359e323aa89fa5bd1d10ada05ea3a01be351ee